src/js/common/schemaform/FormApp.jsx

```diff
@@ -43,14 +56,40 @@ export default class FormApp extends React.Component {
     window.removeEventListener('beforeunload', this.onbeforeunload);
   }
 
+  resetLoading = () => {
+    this.props.setFetchFormStatus(LOAD_STATUSES.notAttempted);
+  }
+
+  resumeForm = () => {
+    this.props.fetchInProgressForm(this.props.formConfig.formId, this.props.formConfig.migrations);
+  }
+
+  goToBeginning = () => {
+    this.props.router.push(this.props.route.pageList[1].path);
+  }
+
   render() {
     const { currentLocation, formConfig, children } = this.props;
     const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
     const isIntroductionPage = trimmedPathname.endsWith('introduction');
     const isConfirmationPage = trimmedPathname.endsWith('confirmation');
     const GetFormHelp = formConfig.getHelp;
     let content;
-    if (!isInProgress(trimmedPathname)) {
+
+    // TODO: As soon as we've redirected, remember to set loadedStatus back to 'not-attempted'
+    if (!formConfig.disableSave && this.props.loadedStatus !== LOAD_STATUSES.notAttempted) {
+      // Show the loading screen instead of the children.
+      // Agh! The prop threading!
```

Comment:
Yeah, that's why I wanted to make `LoadingPage` an actual page. Making an error page probably makes more sense, though.

I remember talking about that, but I was thinking that was covered in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3410. Turns out not so much.